<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Akshay-LAPTOP" failures="1" tests="4" name="Test" time="38.729" errors="0" timestamp="2023-09-09T09:53:03 IST">
  <testcase classname="TestClasses.LoginPageTest" name="verifyLoginTest" time="0.496"/>
  <testcase classname="TestClasses.LoginPageTest" name="verifyLoginTest" time="0.511">
    <failure type="java.lang.AssertionError" message="expected [https://www.saucedemo.com/] but found [https://www.saucedemo.com/inventory.html]">
      <![CDATA[java.lang.AssertionError: expected [https://www.saucedemo.com/] but found [https://www.saucedemo.com/inventory.html]
at TestClasses.LoginPageTest.verifyLoginTest(LoginPageTest.java:34)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- verifyLoginTest -->
  <testcase classname="TestClasses.LoginPageTest" name="verifyLoginTest" time="0.489"/>
  <testcase classname="TestClasses.LoginPageTest" name="verifyLoginTest" time="5.874"/>
</testsuite> <!-- Test -->
